verbcalc/core/translator/symbols.py

```diff
@@ -26,7 +26,7 @@ def __init__(self):
         self._absolutes = ['absolute of', 'absolute value of']
         self._modulo = ['mod', 'modulo']
         self._roots = ['root of']
-        self._questions = ['what is the result of', 'what is', 'calculate']
+        self._questions = ['what is the result of', 'what is', 'calculate', 'when','if', 'result of','addition of','how do you','find','how much','how much is', 'which number is result of', 'subtraction of','what' ]
```

Comment:
Great job, however there are some words that we have to be careful using. 'When' and 'if' are logical connectives and have some deeper meaning that has to be handled separately. I would also opt out from 'addition of', and 'subtraction of', as they suggest the operation that will not be further translated to any symbol. The rest however is very good!
```suggestion
        self._questions = ['what is the result of', 'what is', 'calculate', 'result of', 'how do you', 'find',' how much', 'how much is', 'which number is result of', 'what' ]
```